.github/workflows/node.yml

```diff
@@ -28,9 +28,16 @@ jobs:
     - run: npm ci
     - run: npm run lint
     - run: npm run build --if-present
+    - run: npm run generate:types
     - run: npm test
       env:
         CI: true
+    - name: Install asciidoctor
+      run: sudo apt-get install asciidoctor
+    - name: Install pandoc
+      run: wget https://github.com/jgm/pandoc/releases/download/2.10.1/pandoc-2.10.1-1-amd64.deb && sudo dpkg -i pandoc-2.10.1-1-amd64.deb
+    - name: Convert readme to markdown
+      run: asciidoctor README.adoc -b docbook -o - | pandoc -f docbook -t gfm > README.md
```

Comment:
I experimented with this a bit the other day---I think the move here is:
```
asciidoctor README.adoc -o - | pandoc -f html -t markdown_strict - -o README.md
```

Going through HTML seems to produce better Markdown output (e.g., it includes the Table of Contents and title) than using Docbook as the intermediary.

Reply:
Ah I see, that's probably better than my command since I didn't directly compare the results achieved when using different intermediary formats.